### ISSUE TITLE
javascript timeout

### ISSUE THREAD

**Issue by XMDS** [cleolibrary/CLEO-Redux#22]
This problem occurs in all js scripts. 
At a certain time when the game is running, the script automatically stops and cannot take effect.

```
19:20:57 [ERROR] script FreeSav has timed out after the default timeout of 2 seconds
```
The log file will have the above record

**Comment by yushaer** [comment]
23:56:20 [INFO] CLEO Redux 0.8.3 (x64)
23:56:20 [INFO] Copyright (c) 2021, Seemann. Available under the license agreement at https://re.cleo.li/LICENSE.txt
23:56:20 [INFO] Visit re.cleo.li for news and updates.
23:56:20 [INFO] Host identified as San Andreas: The Definitive Edition (1.0.0.14388)
23:56:20 [INFO] CleoConfig { log_opcodes: false, allow_cs: true, allow_js: true, check_updates: true, permission_level: Lax }
23:56:20 [INFO] Registering custom commands
23:56:20 [INFO] Registering command 0C00 IS_KEY_PRESSED
23:56:20 [INFO] Registering command 0C01 INT_ADD
23:56:20 [INFO] Registering command 0C02 INT_SUB
23:56:20 [INFO] Registering command 0C03 INT_MUL
23:56:20 [INFO] Registering command 0C04 INT_DIV
23:56:20 [INFO] Registering command 0C05 TERMINATE_THIS_CUSTOM_SCRIPT
23:56:20 [WARN] This version of CLEO Redux does not support custom opcodes in main.scm in San Andreas: The Definitive Edition
23:56:55 [INFO] New event received: "init scripts"
23:56:55 [INFO] Started scanning the CLEO directory
23:56:55 [INFO] Found JS script CLEO\SADodgeRoll.js
23:56:55 [INFO] Successfully loaded CLEO\SADodgeRoll.js and created a JS script with the name "SADodge"
23:56:55 [INFO] Found JS script CLEO\Simple-Trainer.js
23:56:55 [INFO] Successfully loaded CLEO\Simple-Trainer.js and created a JS script with the name "Simple-"
23:56:55 [INFO] Found CS script CLEO\menu.cs
23:56:55 [INFO] Loaded CLEO\menu.cs and created a CS script with the name "menu"
00:03:04 [ERROR] script SADodge has timed out after the default timeout of 2 seconds
00:06:14 [ERROR] script SADodge has timed out after the default timeout of 2 seconds

**Comment by yushaer** [comment]
00:39:03 [INFO] CLEO Redux 0.8.3 (x64)
00:39:03 [INFO] Copyright (c) 2021, Seemann. Available under the license agreement at https://re.cleo.li/LICENSE.txt
00:39:03 [INFO] Visit re.cleo.li for news and updates.
00:39:03 [INFO] Host identified as San Andreas: The Definitive Edition (1.0.0.14388)
00:39:03 [INFO] CleoConfig { log_opcodes: false, allow_cs: true, allow_js: true, check_updates: true, permission_level: Lax }
00:39:03 [INFO] Registering custom commands
00:39:03 [INFO] Registering command 0C00 IS_KEY_PRESSED
00:39:03 [INFO] Registering command 0C01 INT_ADD
00:39:03 [INFO] Registering command 0C02 INT_SUB
00:39:03 [INFO] Registering command 0C03 INT_MUL
00:39:03 [INFO] Registering command 0C04 INT_DIV
00:39:03 [INFO] Registering command 0C05 TERMINATE_THIS_CUSTOM_SCRIPT
00:39:03 [WARN] This version of CLEO Redux does not support custom opcodes in main.scm in San Andreas: The Definitive Edition
00:40:08 [INFO] New event received: "init scripts"
00:40:08 [INFO] Started scanning the CLEO directory
00:40:08 [INFO] Found JS script CLEO\Simple-Trainer.js
00:40:08 [INFO] Successfully loaded CLEO\Simple-Trainer.js and created a JS script with the name "Simple-"
00:40:08 [INFO] Found CS script CLEO\menu.cs
00:40:08 [INFO] Loaded CLEO\menu.cs and created a CS script with the name "menu"
00:41:34 [INFO] ["Simple-"]  player is in car
00:41:36 [INFO] ["Simple-"] back
00:41:58 [INFO] ["Simple-"] back
00:44:40 [INFO] ["Simple-"] back
00:46:55 [INFO] New event received: "reload scripts"
00:46:55 [INFO] Script CLEO\Simple-Trainer.js has been disposed
00:46:55 [INFO] Started scanning the CLEO directory
00:46:55 [INFO] Found JS script CLEO\Simple-Trainer.js
00:46:55 [INFO] Successfully loaded CLEO\Simple-Trainer.js and created a JS script with the name "Simple-"
00:46:55 [INFO] Found CS script CLEO\menu.cs
00:46:55 [INFO] Loaded CLEO\menu.cs and created a CS script with the name "menu"
00:47:21 [INFO] ["Simple-"] back
00:48:23 [INFO] ["Simple-"] back
00:48:45 [INFO] ["Simple-"] back
00:57:26 [INFO] ["Simple-"] back
01:00:54 [INFO] ["Simple-"]  player is in car
01:00:55 [INFO] ["Simple-"] back
01:07:43 [INFO] ["Simple-"] back
01:11:10 [INFO] ["Simple-"] back
01:11:35 [ERROR] script Simple- has timed out after the default timeout of 2 seconds
01:17:14 [INFO] New event received: "reload scripts"
01:17:14 [INFO] Script CLEO\Simple-Trainer.js has been disposed
01:17:14 [INFO] Started scanning the CLEO directory
01:17:14 [INFO] Found JS script CLEO\Simple-Trainer.js
01:17:14 [INFO] Successfully loaded CLEO\Simple-Trainer.js and created a JS script with the name "Simple-"
01:17:14 [INFO] Found CS script CLEO\menu.cs
01:17:14 [INFO] Loaded CLEO\menu.cs and created a CS script with the name "menu"
01:18:15 [INFO] New event received: "reload scripts"
01:18:15 [INFO] Script CLEO\Simple-Trainer.js has been disposed
01:18:15 [INFO] Started scanning the CLEO directory
01:18:15 [INFO] Found JS script CLEO\Simple-Trainer.js
01:18:15 [INFO] Successfully loaded CLEO\Simple-Trainer.js and created a JS script with the name "Simple-"
01:18:15 [INFO] Found CS script CLEO\menu.cs
01:18:15 [INFO] Loaded CLEO\menu.cs and created a CS script with the name "menu"
01:21:19 [ERROR] script Simple- has timed out after the default timeout of 2 seconds
01:28:01 [INFO] New event received: "reload scripts"
01:28:01 [INFO] Script CLEO\Simple-Trainer.js has been disposed
01:28:01 [INFO] Started scanning the CLEO directory
01:28:01 [INFO] Found JS script CLEO\Simple-Trainer.js
01:28:01 [INFO] Successfully loaded CLEO\Simple-Trainer.js and created a JS script with the name "Simple-"
01:28:01 [INFO] Found CS script CLEO\menu.cs
01:28:01 [INFO] Loaded CLEO\menu.cs and created a CS script with the name "menu"
01:28:11 [INFO] ["Simple-"] back

**Comment by x87** [comment]
should be fixed in 0.8.4
